packages/fxa-settings/src/styles/custom/UnitRow.scss

```diff
@@ -0,0 +1,59 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+.unit-row {
+  @apply flex flex-wrap px-4 pt-7 pb-8;
+
+  &-header {
+    @apply w-full mb-1;
+  }
+
+  &-header-value {
+    @apply w-full;
+  }
+
+  &-cta-container {
+    @apply w-full;
+  }
+
+  &-hr {
+    @apply mx-5 border-grey-100;
+  }
+}
+
+.cta {
```

Comment:
For consistency this could be called `unit-row-cta` for now, I don't really want to call it a `button` because sometimes this is a link - the design guide calls it `button-neutral`.  We'll also be using this in the `Connected Services` section and when we do that, `cta` could be in its own file and makes sense. We could call the "Disable" buttons / "Delete account" link as something like `cta-warning`.